src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -501,6 +501,16 @@ protected function _list( $_, $assoc_args ) {
 
 		$all_items = $this->get_all_items();
 
+		// Only show plugins that are recently active.
```

Comment:
This comment isn't necessary and can be removed. It simply duplicates the code.

Reply:
Removed in https://github.com/wp-cli/extension-command/pull/383/commits/805760d37ffd2fc84aa4f7e874fcbf3944af9d79, but I didn't get what you meant here.